test/metabase/api/collection_test.clj

```diff
@@ -1380,6 +1380,60 @@
                        (mt/user-http-request :rasta :put 403 (str "collection/" (u/the-id collection-a))
                                              {:parent_id (u/the-id collection-c)})))))))))))
 
+;;; +----------------------------------------------------------------------------------------------------------------+
+;;; |                          GET /api/collection/root|:id/timelines                                                |
+;;; +----------------------------------------------------------------------------------------------------------------+
+
+(defn- timelines-request
+  [collection include-events?]
+  (if include-events?
+    (mt/user-http-request :rasta :get 200 (str "collection/" (u/the-id collection) "/timelines") :include "events")
+    (mt/user-http-request :rasta :get 200 (str "collection/" (u/the-id collection) "/timelines"))))
+
+(defn- timeline-names [timelines]
+  (->> timelines (map :name) set))
+
+(defn- event-names [timelines]
+  (->> timelines (mapcat :events) (map :name) set))
+
+(deftest timelines-test
+  (testing "GET /api/collection/root|id/timelines"
+    (mt/with-temp* [Collection [coll-a {:name "Collection A"}]
+                    Collection [coll-b {:name "Collection B"}]
+                    Collection [coll-c {:name "Collection C"}]]
+    (mt/with-temp* [Timeline [tl-a {:name          "Timeline A"
```

Comment:
You can just use a single `with-temp*` block, no need for the two nested ones.

Reply:
Good catch, thanks. Fixed here and in cards test too